community-operators/microcks/microcks-operator.v0.1.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,197 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: microcks-operator.v0.1.0
+  namespace: REPLACE_NAMESPACE
+  annotations:
+    categories: "Mocking & Testing"
+    certified: "false"
+    containerImage: quay.io/microcks/microcks-ansible-operator:0.1.0
+    createdAt: 2019-03-14
+    support: microcks.github.io
+    alm-examples: |-
+      [
+        {
+          "apiVersion": "microcks.github.io/v1alpha1",
+          "kind": "MicrocksInstall",
+          "metadata": {
+            "name": "my-microcksinstall"
+          },
+          "spec": {
+            "name": "my-microcksinstall",
+            "version": "0.7.1",
+            "microcks": {
+              "replicas": 2
+            },
+            "postman": {
+              "replicas": 2
+            },
+            "keycloak": {
+              "install": true,
+              "persistent": true,
+              "volumeSize": "1Gi"
+            },
+            "mongodb": {
+              "install": true,
+              "persistent": true,
+              "volumeSize": "2Gi"
+            }
+          }
+        }
+      ]
+spec:
+  displayName: microcks-operator
```

Comment:
This will be the name displayed in the UI. It could be `Microcks Operator` for example. The name you choose is up to you.

Reply:
Thanks for suggestion. Aligned to it in commit e49983af1e6a18db2c377218a87eab53b9235bd7